javascript/javascript1/week2/event-application1.js

```diff
@@ -0,0 +1,15 @@
+function myFunction(numdays) {
+    const alldays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+    d = new Date()
+    today = d.getDay();
+    let num = (today+numdays) % 7
+    return "today is " + alldays[today] + " and the even will be held on " + alldays[num] + "."
+};
+
+let a = myFunction(4)
```

Comment:
what would happen if I call this function like this 

`const a = myFunction(-8)`